src/Templates/src/templates/maui-blazor/.template.config/template.json

```diff
@@ -53,7 +53,7 @@
         "type": "parameter",
         "dataType": "string",
         "replaces": "MS_EXT_VERSION",
-        "defaultValue": "MS_EXT_VERSION_VALUE"
+        "defaultValue": "MS_EXT_LOG_DEBUG_VERSION_VALUE"
```

Comment:
Same thing for regular MAUI template? https://github.com/dotnet/maui/blob/main/src/Templates/src/templates/maui-blazor/.template.config/template.json#L52-L56

Reply:
I think we should also rename the `msExtensionsVersion` variable name to specifically `msExtensionsLoggingDebugVersion` so that people don't assume the versions of all MS Extensions things are the same (sometimes they are, sometimes they aren't).

Reply:
Fixed